spyder_env_manager/spyder/widgets/main_widget.py

```diff
@@ -206,6 +213,34 @@ def setup(self):
             triggered=self._message_export_environment,
         )
 
+        exclude_dependency_action = self.create_action(
+            SpyderEnvManagerWidgetActions.ToggleExcludeDependency,
+            text=_("Exclude dependency packages"),
+            tip=_("Exclude dependency packages"),
+            toggled=True,
+            triggered=self.update_packages,
+            option=SpyderEnvManagerWidgetActions.ToggleExcludeDependency,
+            initial=self.get_conf(
+                SpyderEnvManagerWidgetActions.ToggleExcludeDependency, True
+            ),
```

Comment:
Is it really necessary to set an initial value here? I mean, the action is already connected to the `ToggleExcludeDependency` option and its value is set in `spyder_env_manager/spyder/config.py`, so I don't see the need for this.

In addition, the problem with setting default values through `get_conf` is that later you can't change them in `config.py` and bumping `CONF_VERSION`. So, if possible, I think this should be removed.

Finally, if the action doesn't set a default value according to the option associated to it, that's a bug in Spyder and we need to fix it there.

Reply:
Basically I did this to prevent the test from failing since currently it only tests the `main_widget` and fails if no default value is defined since no plugin is initialized I think (see https://github.com/spyder-ide/spyder-env-manager/actions/runs/4236044448/jobs/7360400388).

I could take the default value from `spyder_env_manager/spyder/config.py` and set it in the calls. That should prevent the config synchornization problem, right? Or maybe do you have other suggestion to handle this?

Reply:
I see. The thing is the current tests are quite simple (just initializing `SpyderEnvManagerWidget` and showing it) but I saw that you improved them in PR #14 by monkey-patching its `get_conf` method and making it return the default values set in `spyder_env_manager/spyder/config.py`.

So, I propose to disable the tests in this PR to remove setting these default values, rebase PR #14 afterwards and run the tests there, which I think are more complete than the ones here.

Reply:
I think the monkeypatch I was doing in PR #14 needs changes (I'm using there the default value passed to the `get_conf`). However using the `CONF_DEFAULTS` is a good idea :+1: I did that here and removed the default values from the `get_conf` calls in the toggle actions definitions in the latest commit. With the config sync should not be a problem right?